tools/CoqMakefile.in

```diff
@@ -668,6 +667,7 @@ uninstall:: .filestoinstall
 	 echo RMDIR "$(COQLIBINSTALL)/$$df/" &&\
 	 (rmdir "$(COQLIBINSTALL)/$$df/" 2>/dev/null || true); \
 	done
+	rm -f .filestoinstall
```

Comment:
```suggestion
	@rm -f .filestoinstall
```
Since `.filestoinstall` is basically an internally used file, do we really want to show this command? It might confuse people.

Reply:
indeed